core/src/main/java/io/fabric8/maven/core/util/ProcessUtil.java

```diff
@@ -123,6 +127,14 @@ protected static void processOutput(Logger log, InputStream inputStream, Functio
     public static File findExecutable(Logger log, String name) {
         List<File> pathDirectories = getPathDirectories(log);
         for (File directory : pathDirectories) {
+            if( isWindows() ) {
+                for (String extension : new String[]{".exe", ".bat", ".cmd"}) {
+                    File file = new File(directory, name+extension);
+                    if (file.exists() && file.isFile()) {
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1105127/19240607/74a78036-8f0b-11e6-9ec6-8ed8052453fc.png)
there is a "read & execute" permission so I suppose that it is the one used